packages/@aws-cdk/aws-appmesh/lib/shared-interfaces.ts

```diff
@@ -238,7 +263,17 @@ class VirtualServiceBackend extends Backend {
       virtualServiceBackend: {
         virtualService: {
           virtualServiceName: this.virtualService.virtualServiceName,
-          clientPolicy: this.clientPolicy?.bind(_scope).clientPolicy,
+          clientPolicy: this.tlsClientPolicy !== undefined ?
+            {
```

Comment:
Indentation should be different here:

```suggestion
          clientPolicy: this.tlsClientPolicy !== undefined
            ? {
```

Reply:
Addressed